autoread.js

```diff
@@ -113,4 +115,21 @@ function handleCheckbox(currentLine){
 
 function handleSublist(currentLine){
 	return Array.of(...currentLine.childNodes).filter(x=>x.nodeName != "OL" && x.nodeName != "UL").map(x=>x.textContent).join('');
+}
+
+function orientationCheck(){
+		let sBar = document.getElementById("sidebar");
+		let sBox = document.getElementById("speechbox");
+		
+		if(window.innerHeight + 100 > window.innerWidth){
+			//portrait mode
+			sBar.style.position="fixed";
+			sBar.style.bottom="1em";
+			sBox.style.border="2px solid black";
```

Comment:
I don't love changing the sidebar style here. I'm not sure what to do about it, though. Brainstorming ideas (may be bad):
-Maybe this swaps out the sidebar class (styled in css document) from "sideways" to "bottom"?
-maybe remove speech box from sidebar and leave it floating?